src/test/java/org/usfirst/frc/team1294/robot/vision/VisionProcessingTests.java

```diff
@@ -0,0 +1,133 @@
+package org.usfirst.frc.team1294.robot.vision;
+
+import org.hamcrest.core.Is;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.opencv.core.Mat;
+import org.opencv.core.MatOfInt;
+import org.opencv.core.Point;
+import org.opencv.core.Scalar;
+import org.opencv.imgcodecs.Imgcodecs;
+import org.opencv.imgproc.Imgproc;
+import org.usfirst.frc.team1294.robot.subsystems.CameraSubsystem;
+
+import java.io.*;
+
+import static org.hamcrest.core.Is.is;
+import static org.junit.Assert.assertThat;
+
+public class VisionProcessingTests {
+
+  @BeforeClass
+  public static void loadNativeOpenCvLibraries() throws Exception {
+    try {
+      String osname = System.getProperty("os.name");
+      String resname = "/opencv/";
+      if (osname.startsWith("Windows"))
+        resname += "Windows/" + System.getProperty("os.arch") + "/";
+      else
+        resname += osname + "/" + System.getProperty("os.arch") + "/";
+
+      if (System.getProperty("os.name").startsWith("Linux") && new File("/usr/lib/arm-linux-gnueabihf").exists()) {
```

Comment:
Looks like Travis builds are failing because it's looking for `amd64` builds for "`linux`". `.startsWith("Linux")` will return false because "Linux" != "linux". Also, we don't appear to have `amd64` builds for Travis...